src/Login.java

```
@@ -15,20 +18,45 @@ public static boolean verification() {
         } catch (FileNotFoundException e) {
             e.getStackTrace();
         }
-        while (scan.hasNext()) {
+        while (Objects.requireNonNull(scan).hasNext()) {
             username = scan.next();
             password = scan.next();
         }
+        scan.close();
         System.out.println("Please log in:");
         System.out.print("Username: ");
         String inputUsername = input.next();
         System.out.print("Password: ");
         String inputPassword = input.next();
 
-        if (inputUsername.equals(username) && inputPassword.equals(password)) {
-            return true;
+        return inputUsername.equals(username) && inputPassword.equals(password);
+    }
+
+    public static void changeLogin() {
+        Scanner input = new Scanner(System.in);
+        File file = new File("login.txt");
+        if (file.delete()) {
+            System.out.println("Change Username and Password.");
+            System.out.print("Username: ");
+            String username = input.next();
+            System.out.print("Password: ");
+            String password = input.next();
+            try {
+                if (file.createNewFile()) {
+                    FileWriter output = new FileWriter(file);
+                    output.write(username);
+                    output.write(" ");
+                    output.write(password);
+
+                    output.close();
+                } else {
+                    System.out.println("Something went wrong.");
+                }
+            } catch (IOException e) {
```

do not swallow the exception. Or at least if you do, print the error message and return a false response. Do not use e.printStackTrace(). It is not recommended.